docs/changelog.md

```diff
@@ -2,6 +2,7 @@
 
 Versions follow [Semantic Versioning](https://semver.org/): `<major>.<minor>.<patch>`.
 
+* Add dialog boxes.
```

Comment:
Need to add the types of dialog boxes supported.

Reply:
Ready